config/manager/manager.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     control-plane: controller-manager
     controller-tools.k8s.io: "1.0"
   ports:
-  - port: 443
+    - port: 443
```

Comment:
we should standardize on the formatting of lists in YAML, I feel like we go back and forth between

```yaml
list:
- item1
- item2
```
and
```yaml
list:
  - item 1
  - item 2
```

I recommend the first approach since that is what I commonly see in k8s.io examples, but if there's an actual reason for this style that's fine too.

Reply:
It's RH's YAML extension in VSCode auto-formatting, and I wasn't paying attention.

Reply:
Disabled